packages/api/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "module": "esnext",
+    "target": "esnext",
+    "sourceMap": false,
+    "moduleResolution": "Node",
+    "skipLibCheck": true,
```

Comment:
maybe use false here if we can (or have the stronger checks)


it probably misses the checks for TypeScript as well (and optional unit tests that should be run as part of the root package.json file)

Reply:
Removed! We will see :)